docs/index.md

```diff
@@ -124,76 +132,88 @@ spec:
 
+## I am seeing this error message in the logs of the Ingress-NGINX controller: "ingress class annotation is not equal to the expected by Ingress Controller". Why ?
 
-## I am seeing this error message in the logs of the Ingress-NGINX controller "ingress class annotation is not equal to the expected by Ingress Controller". Why ?
-- It is highly likely that you will also see the name of the ingress resource in the same error message. This error messsage has been observed on use the deprecated annotation, to spec the ingressClass, in a ingress resource manifest. It is recommended to use the ingress.spec.ingressClassName field, of the ingress resource, to spec the name of the ingressClass of the ingress resource being configured.
+- It is highly likely that you will also see the name of the ingress resource in the same error message. This error messsage has been observed on use the deprecated annotation (`kubernetes.io/ingress.class`) in a Ingress resource manifest. It is recommended to use the `.spec.ingressClassName` field of the Ingress resource, to specify the name of the IngressClass of the Ingress you are defining.
 
 ## How to easily install multiple instances of the ingress-NGINX controller in the same cluster ?
 - Create a new namespace
   ```
   kubectl create namespace ingress-nginx-2
   ```
-- Use helm to install the additional instance of the ingress controller
-- Ensure you have helm working (refer to helm documentation)
-- We have to assume that you have the helm repo for the ingress-NGINX controller already added to your helm config. But, if you have not added the helm repo then you can do this to add the repo to your helm config;
+- Use Helm to install the additional instance of the ingress controller
+- Ensure you have Jelm working (refer to the [Helm documentation](https://helm.sh/docs/))
+- We have to assume that you have the helm repo for the ingress-NGINX controller already added to your Helm config. But, if you have not added the helm repo then you can do this to add the repo to your helm config;
   ```
   helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx
   ```
 - Make sure you have updated the helm repo data;
   ```
   helm repo update
   ```
-- Now you install the additional instance of the ingress-NGINX controller like this ;
+- Now, install an additional instance of the ingress-NGINX controller like this ;
   ```
   helm install ingress-nginx-2 ingress-nginx/ingress-nginx  \
   --namespace ingress-nginx-2 \
-  --set controller.ingressClassResource.name=nginx-2 \
-  --set controller.ingressClassResource.controllerValue="k8s.io/ingress-nginx-2" \
+  --set controller.ingressClassResource.name=nginx-two \
+  --set controller.ingressClassResource.controllerValue="example.com/ingress-nginx-2" \
```

Comment:
```suggestion
  --set controller.ingressClassResource.controllerValue="example.com/ingress-nginx-two" \
```

Reply:
Actually should be `example.com/ingress-nginx2` to match earlier text.